Teams docs (#117)
tinacms/tinacms-site: content/docs/teams/introduction.md

---
title: Introduction
id: /docs/teams/introduction
prev: null
next: /docs/teams/cli/introduction
---

> When teams are editing a site with TinaCMS, typically a **cloud editing environment** with TinaCMS be hosted somewhere (so that editors do not need to install the site in a local development environment).

Reviewer: ```suggestion
When teams are editing a site with TinaCMS, typically a [**cloud editing environment**](https://tinacms.org/blog/editing-on-the-cloud/) with TinaCMS be hosted somewhere (so that editors do not need to install the site in a local development environment).
```

---
title: Introduction
id: /docs/teams/introduction
prev: null
next: /docs/teams/gatsby/introduction
---

> When it's time to get your editors editing a TinaCMS site, you will likely want to setup a **cloud editing environment** so that editors can edit the site's content from the cloud. The **cloud editing environment** is separate from your production site. It is a staging environment which runs your site in dev-mode, in which TinaCMS will be accesible.

Reviewer 2: ```suggestion
When it's time to get your editors editing a TinaCMS site, you will likely want to setup a **cloud editing environment** so that editors can edit the site's content from the cloud. The **cloud editing environment** is separate from your production site. It is a server which runs your site in dev-mode from which TinaCMS will be accessible.
```

Once a cloud editing environment has been setup for your site, **Tina Teams** can be used to manage which users can access the cloud editing environment.

## Get Started

Let's walk you through all the steps required to get your editors editing in the cloud.

### 1) Deploy your Cloud Editing Environment

Part of what makes Tina great is that it gives the developer control. It’s important for us to extend this control into the Cloud editing experience. For this reason, we’ve designed it so that you can host your cloud editing environment wherever you like (Gatsby Cloud, Heroku, Digital Ocean, etc).

Check out our guide to [setting up a cloud editing environment](/blog/using-tinacms-on-gatsby-cloud) on Gatsby Cloud

### 2) Add Tina Teams package to your site

Now that our cloud editing environment is live, we will want to manage who can access this environment. We also need a way to identify each user so that we can attribute saves made through TinaCMS with it's author.

> E.g: If i am using the **gatsby-tinacms-git** plugin, when a user hits "save" it will create a commit with the author set to the logged-in user.

Reviewer 2: "e.g" is "example given"

"i.e." is "that is to say"

[ref](https://www.grammarly.com/blog/know-your-latin-i-e-vs-e-g/)

```suggestion
i.e.: If i am using the **gatsby-tinacms-git** plugin, when a user hits "save" it will create a commit with the author set to the logged-in user.
```
